install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```
@@ -307,7 +307,7 @@ spec:
           mountPath: /var/lib/cilium/bgp
           readOnly: true
         {{- end }}
-        {{- if and .Values.hubble.enabled .Values.hubble.tls.enabled (hasKey .Values.hubble "listenAddress") }}
+        {{- if and .Values.hubble.enabled .Values.hubble.tls.enabled (hasKey .Values.hubble "listenPort") }}
```

I think theses were missed and still use `listenPort`.

```suggestion
        {{- if and .Values.hubble.enabled .Values.hubble.tls.enabled (hasKey .Values.hubble "listenAddress") }}
```